website/docs/matchers/Array.mdx

```diff
@@ -62,13 +62,13 @@ Use `.toIncludeAnyMembers` when checking if an `Array` contains any of the membe
 
 ### .toIncludeSameMembers([members])
 
-Use `.toIncludeSameMembers` when checking if two arrays contain equal values, in any order.
+Use `.toIncludeSameMembers` when checking if two arrays contain members in exact match, in any order.
```

Comment:
I'm not sure this is clearer. The matcher is using the `equals` function, which is why the original was worded the way it was.